doc/python_reference.rst

```diff
@@ -263,6 +263,10 @@ Visualization
    plot_alignment
    snapshot_brain_montage
    plot_arrowmap
+   set_3d_backend
+   get_3d_backend
+   use_3d_backend
+   use_test_3d_backend
```

Comment:
don't expose to API something we only use for testing. Remove use_test_3d_backend and I would even call it _use_test_3d_backend